build.sbt

```diff
@@ -48,12 +48,12 @@ dependencyOverrides ++= Seq(
   "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
   "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
   "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
-  "software.amazon.awssdk" % "netty-nio-client" % "2.20.26",
+  "software.amazon.awssdk" % "netty-nio-client" % "2.26.25",
   "org.json" % "json" % "20231013",
-  "io.netty" % "netty-handler" % "4.1.94.Final",      // SNYK-JAVA-IONETTY-5725787
-  "io.netty" % "netty-codec-http2" % "4.1.100.Final", // SNYK-JAVA-IONETTY-5953332
```

Comment:
Presumably these have been subsumed elsewhere? Or perhaps not used any more anyway?

Reply:
Yes I bumped the library that included them so they are not needed any more (tested with snyk)